pkg/gofr/config/godotenv.go

```diff
@@ -35,29 +35,33 @@ func (e *EnvLoader) read(folder string) {
 		env          = e.Get("APP_ENV")
 	)
 
-	if env != "" {
-		overrideFile = fmt.Sprintf("%s/.%s.env", folder, env)
+	err := godotenv.Load(defaultFile)
+	if err != nil {
+		e.logger.Warnf("Failed to load config from file: %v, Err: %v", defaultFile, err)
+	} else {
+		e.logger.Infof("Loaded config from file: %v", defaultFile)
 	}
 
-	// If 'APP_ENV' is set to x, then GoFr will try to read '.x.env' file from configs directory, if failed it will read
-	// the default file '.env'.
-	// If 'APP_ENV' is not set , GoFr will first try to read '.local.env', if failed then it
-	// will read default env file.
-	err := godotenv.Load(overrideFile)
-	if err != nil {
-		if env != "" {
+	switch env {
+	case "":
+		// If 'APP_ENV' is not set , GoFr will first read '.env' and override any configs present in '.local.env'
+		err := godotenv.Overload(overrideFile)
+		if err != nil {
 			e.logger.Warnf("Failed to load config from file: %v, Err: %v", overrideFile, err)
```

Comment:
warn message is not needed in this case, we can only give INFO if we are reading a file.

Reply:
sure will replace it by `Debug` log